server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -109,7 +109,7 @@ export default class ServiceProviderReferralsController {
       res.locals.user.token.accessToken,
       SPDashboardType.MyCases
     )
-    this.renderDashboardWithoutPagination(req, res, referralsSummary, 'My cases')
+    await this.renderDashboardWithoutPagination(req, res, referralsSummary, 'My cases')
```

Comment:
why we need to change this to async?

Reply:
> why we need to change this to async?. Same question holds good for other places as well

Because we are calling an async function in ReferenceDataService to load the what's new banner content from a csv from `ControllerUtils`.  `MakeAReferralController` also uses the ReferenceDataService and all the functions are async.